<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateRoomDialog</class>
 <widget class="QDialog" name="CreateRoomDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>264</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,8">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Rooom ID:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="roomIdLineEdit">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,8">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Room Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="roomNameLineEdit">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,8">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Secret:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="secretLineEdit">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="2,8">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>PIN:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pinLineEdit">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="2,8">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Permanent:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QRadioButton" name="permanentRadioButtonYes">
          <property name="text">
           <string>YES</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="permanentRadioButtonNo">
          <property name="text">
           <string>NO</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="2,8">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Private:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QRadioButton" name="privateRadioButtonYes">
          <property name="text">
           <string>YES</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="privateRadioButtonNo">
          <property name="text">
           <string>NO</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CreateRoomDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CreateRoomDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
